app/core/db.py

```
@@ -1,18 +1,40 @@
-engine = create_engine(
-    SQLALCHEMY_DATABASE_URL,
-    pool_pre_ping=True,
-)
+USE_CLOUD_SQL = os.getenv("USE_CLOUD_SQL", "0") == "1"
+
+def get_engine():
+    if USE_CLOUD_SQL:
+        # Cloud SQL 필수 환경 변수 검증
+        instance_name = os.getenv("INSTANCE_CONNECTION_NAME")
+        if not all([instance_name, settings.DB_USER, settings.DB_PASSWORD, settings.DB_NAME]):
+            raise ValueError("Cloud SQL 사용 시 INSTANCE_CONNECTION_NAME, DB_USER, DB_PASSWORD, DB_NAME이 모두 필요합니다.")
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)